crates/astria-grpc-mock/src/mock.rs

```diff
@@ -45,6 +46,13 @@ impl Mock {
         }
     }
 
+    #[must_use = "a mock must be mounted on a server to be useful"]
+    pub fn up_to_n_times(mut self, n: u64) -> Self {
```

Comment:
This is the original wiremock functionality that we hadn't yet ported over right?

Reply:
Yup, indeed.